packages/content-management/favorites/favorites_server/src/favorites_saved_object.ts

```diff
@@ -21,13 +21,19 @@ const schemaV1 = schema.object({
   favoriteIds: schema.arrayOf(schema.string()),
 });
 
+export const favoritesSavedObjectName = 'favorites';
+
 export const favoritesSavedObjectType: SavedObjectsType = {
-  name: 'favorites',
+  name: favoritesSavedObjectName,
   hidden: true,
   namespaceType: 'single',
   mappings: {
     dynamic: false,
-    properties: {},
+    properties: {
```

Comment:
We didn't need these mappings for [the feature](https://github.com/elastic/kibana/pull/189285), but, unfortunately, I didn't know that we would need them for the snapshot telemetry. I am not sure if there is another better way


the object schema stays the same, but we add existing fields to the mapping

Reply:
It's been my experience as well that it's sometimes necessary to add mappings just for the sake of being able to collect snapshot telemetry.